sklearn/pipeline.py

```diff
@@ -80,11 +80,11 @@ class Pipeline(_BaseComposition):
         estimator.
 
     memory : str or object with the joblib.Memory interface, default=None
-        Used to cache the fitted transformers of the pipeline. By default,
-        no caching is performed. If a string is given, it is the path to
-        the caching directory. Enabling caching triggers a clone of
-        the transformers before fitting. Therefore, the transformer
-        instance given to the pipeline cannot be inspected
+        Used to cache the fitted transformers of the pipeline. Currently,
+        the last step will never be cached. By default, no caching is performed.
```

Comment:
```suggestion
        Used to cache the fitted transformers of the pipeline. The last step will never be cached, even if it is a transformer. By default, no caching is performed.
```

Reply:
You can apply the same changes in below.

Reply:
@glemaitre I think I implemented your suggestion consistently across the three touched locations, now.